host_applications/linux/libs/sm/user-vcsm.c

```diff
@@ -273,13 +273,22 @@ int vcsm_init_ex( int want_export, int fd )
       goto out; /* VCSM already opened. Nothing to do. */
    }
 
+   if (fd != -1)
+   {
+      vcsm_handle = dup(fd);
+
+      // FIXME: Sanity check which device that the fd actually relates to.
+      // For now we have to guess based on whether export is requested.
+      // (the main use case is from Chromium which will be requesting export).
+      if (want_export)
+         using_vc_sm_cma = 1;
+
+      goto out;
```

Comment:
Is it OK to bypass the assignment to vcsm_page_size?

Reply:
Nope. Fixed.